### PR TITLE
[dv] Updated SIM_FW_SIMULATED V2 checklist wording

### PR DESCRIPTION
**Pull request by sriyerg**
This commit clarifies what this checklist item indicates.

Signed-off-by: Srikrishna Iyer <sriyer@google.com>

### CODE REVIEW

File: doc/project/checklist.md [hunk 1]
```diff
@@ -256,6 +256,11 @@ A testplan has been written (in HJson format) indicating:
 - The functional coverage plan captured as a list of covergroups, with a description highlighting which feature is expected to be covered by each covergroup.
   It may optionally contain additional details such as coverpoints and crosses of individual aspects of the said feature that is covered.
 
+If the DUT has a CPU for which a ROM firmware is developed (burnt-in during manufacturing):
+- A detailed ROM firmware testplan has been written to adequately verify all functions of the ROM firmware in a pre-silicon simulation environment (i.e. a DV testbench).
+- The testing framework may validate each functionality of the ROM firmware discreetly as an individual unit test.
+- Depending on the ROM firmware development progress, this may be postponed for V2.
+
```
Comment (sriyerg) [inline doc/project/checklist.md:263]: Additionally, in V1 stage, clarify that the testplan must also include the verification of ROM firmware functions if possible.

**Review by timothytrippel — APPROVED** [review]

**Review by weicaiyang — APPROVED** [review]
LGTM

File: doc/project/checklist.md [hunk 2]
```diff
@@ -256,6 +256,11 @@ A testplan has been written (in HJson format) indicating:
 - The functional coverage plan captured as a list of covergroups, with a description highlighting which feature is expected to be covered by each covergroup.
   It may optionally contain additional details such as coverpoints and crosses of individual aspects of the said feature that is covered.
 
+If the DUT has a CPU for which a ROM firmware is developed (burnt-in during manufacturing):
+- A detailed ROM firmware testplan has been written to adequately verify all functions of the ROM firmware in a pre-silicon simulation environment (i.e. a DV testbench).
+- The testing framework may validate each functionality of the ROM firmware discreetly as an individual unit test.
```
Comment (rswarbrick) [inline doc/project/checklist.md:261]: Nit: This should be "discretely" (at the moment, you're asking the testing framework to keep it on the down low :-p)

**Review by rswarbrick — APPROVED** [review]
Spelling nit, but otherwise looks good to me.

**Review by matutem — APPROVED** [review]

File: doc/project/checklist.md (hunk 2 above)
Comment (sriyerg) [reply doc/project/checklist.md]: lol, fixed :)

**Pull request merged**